S3Scanner/__main__.py

```diff
@@ -109,6 +110,7 @@ def scan_single_bucket(s3service, anons3service, do_dangerous, bucket_name):
             s3service.check_perm_write_acl(b)
 
     print(f"{b.name} | bucket_exists | {b.get_human_readable_permissions()}")
+    return b.asdict() if json else None
```

Comment:
Instead of printing in the method and then possibly returning a dict, let's always return a dict and then either print or write JSON with the output